label_sync/labels.yaml

```diff
@@ -338,13 +338,13 @@ default:
       prowPlugin: shrug
       addedBy: humans
 repos:
-  tektoncd/catalog:
   tektoncd/cli:
-    - color: 3f6eb5
-      description: Issues related to user-studies
-      name: user-study
-      target: issues
-      addedBy: humans
+    labels:
+      - color: 3f6eb5
+        description: Issues related to user-studies
+        name: user-study
+        target: issues
+        addedBy: humans
```

Comment:
question - does this mean you are doing user studies @vdemeester ? we've got an `area/user-study` label in pipelines also (https://github.com/tektoncd/pipeline/labels/area%2Fuser-study) but i dont think we actually need it anymore

Reply:
Ahh 😅 as it was there before I kept it 😇

Reply:
ah kk ill make a note and see if can get around to removing them XD